pom.xml

```diff
@@ -69,7 +69,13 @@
     <dependency>
       <groupId>com.github.cbioportal.cbioportal</groupId>
 	  <artifactId>core</artifactId>
-      <version>v3.6.2</version>
+      <version>24bc8b56d73cacb33a1601e3a60160ceff2fa13d</version>
+      <exclusions>
+          <exclusion>
+              <groupId>org.apache.logging.log4j</groupId>
+              <artifactId>log4j-core</artifactId>
+          </exclusion>
+      </exclusions>
```

Comment:
this was excluded before, but i included it when we made this a core only dep - excluding it again